nncf/tensorflow/graph/model_transformer.py

```diff
@@ -61,6 +62,7 @@ def __init__(self, model):
             list(get_custom_objects(model).items()) + list(get_nncf_custom_objects().items())
         )
         self._name_mapping = {}
+        self._weight_operations_for_insertion_per_layer_name = defaultdict(list)
```

Comment:
It looks like the merge artifacts

Reply:
Indeed, had a wrong version of a previous commit applied on top of the one that went to develop.